webapp/channels/src/components/widgets/menu/menu_header.tsx

```diff
@@ -11,20 +10,14 @@ type Props = {
     onClick?: () => void;
 }
 
-/**
- * @deprecated Use the "webapp/channels/src/components/menu" instead.
- */
```

Comment:
please bring the deprecated notice

Reply:
Due to this deprecated notice I am not able to to export the menuheader component at last that why I had to remove it. Please suggest the solution for it. #

Reply:
Try to put the deprecated notice just before the export.